src/schoology-extractor/schoology_extractor/main.py

```diff
@@ -3,76 +3,128 @@
 # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 # See the LICENSE and NOTICES files in the project root for more information.
 
+import logging
```

Comment:
I know this ticket is not primarily for exception handling. However, can you add a try/catch for this script so that any exceptions will be logged using the logger instead of going to standard error output?

Reply:
✅